### PR TITLE
CUMULUS-1686: AMI ID tfvar required

### PR DESCRIPTION
**Pull request by nemreid**
**Summary:** Summary of changes

Addresses [CUMULUS-1686: Remove default ECS AMI from TF](https://bugs.earthdata.nasa.gov/browse/CUMULUS-1686)

## Changes

  - Changed `ecs_cluster_instance_image_id` to be a required variable of the `cumulus` module. The default was not available across accounts and regions, nor outside of NGAP and has been removed.

## PR Checklist

- [ ] Update CHANGELOG
- [ ] Unit tests
- [ ] Adhoc testing
- [ ] Integration tests

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -36,6 +36,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - **CUMULUS-1626**
   - Updates Cumulus to use node10/CMA 1.1.2 for all of its internal lambdas in prep for AWS node 8 EOL
 
+- **CUMULUS01686
```
Comment (laurenfrederick) [inline CHANGELOG.md]:
```suggestion
- **CUMULUS-1686**
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -36,6 +36,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
...
+  - Changed `ecs_cluster_instance_image_id` to be a required variable of the `cumulus` module. The default was not available across accounts and regions, nor outside of NGAP and has been removed.
```
Comment (laurenfrederick) [inline CHANGELOG.md]: This is breaking-ish, right? Should we have a note at the top of the changelog for this?

Comment (nemreid) [reply CHANGELOG.md]: Good point. I'll update.

Comment (nemreid) [reply CHANGELOG.md]: c22dc21

**Pull request merged**